src/submitter.py

```diff
@@ -270,14 +270,9 @@ def fetch_dumps(client, bucket, crash_id):
 
 
 COLLECTOR_KEYS_TO_REMOVE = [
-    "collector_notes",  # version 1
-    "dump_checksums",  # version 1
-    "metadata",  # version 2
-    "MinidumpSha256Hash",  # version 1
-    "payload_compressed",  # version 1
-    "payload",  # version 1
+    "metadata",
     "submitted_timestamp",
-    "version",  # version 2
```

Comment:
One nice thing about v2 of the schema is that the bulk of the things the collector adds are in "metadata", so we can just remove that in the submitter and we don't have to keep updating the submitter every time the collector adds something new.